.github/workflows/pull_request.yaml

```diff
@@ -131,23 +108,45 @@ jobs:
           echo "" >> pr_message.txt
           humctl diff sets env/${{ env.ENVIRONMENT_ID }} env/${{ env.BASE_ENVIRONMENT }} \
               --token ${{ secrets.HUMANITEC_TOKEN }} \
-              --org ${{ vars.HUMANITEC_ORG }} \
-              --app ${{ vars.APP_NAME }} -o json >> pr_message.txt
+              --org ${{ vars.HUMANITEC_ORG_ID }} \
+              --app ${{ env.APP_NAME }} -o json >> pr_message.txt
           echo "" >> pr_message.txt
           echo '```' >> pr_message.txt
           echo "" >> pr_message.txt
           echo "</details>" >> pr_message.txt
 
-          echo "<details><summary>Score Output</summary>" >> pr_message.txt
-          echo "" >> pr_message.txt
-          echo "### Score Output:" >> pr_message.txt
-          echo '```json' >> pr_message.txt
-          echo "" >> pr_message.txt
-          cat score_output.json >> pr_message.txt
-          echo "" >> pr_message.txt
-          echo '```' >> pr_message.txt
-          echo "" >> pr_message.txt
-          echo "</details>" >> pr_message.txt
+          if [ "$DEPLOYMENT_ERRORS" = "[]" ]; then
+            echo "<details><summary>Active Resources Usage</summary>" >> pr_message.txt
+            echo "" >> pr_message.txt
+            echo "### Active Resources Usage:" >> pr_message.txt
+            echo '```none' >> pr_message.txt
+            echo "" >> pr_message.txt
+            humctl resources active-resource-usage \
+                --env ${{ env.ENVIRONMENT_ID }} \
+                --token ${{ secrets.HUMANITEC_TOKEN }} \
+                --org ${{ vars.HUMANITEC_ORG }} \
+                --app ${{ vars.APP_NAME }} >> pr_message.txt
+            echo "" >> pr_message.txt
+            echo '```' >> pr_message.txt
+            echo "" >> pr_message.txt
+            echo "</details>" >> pr_message.txt
+          fi
+          if [ "$DEPLOYMENT_ERRORS" = "[]" ]; then
```

Comment:
One small nit. If both the if conditions are same, can we have them in one if condition. Otherwise PR LGTM.